packages/brokeneck-react/src/components/EntityFields.js

```diff
@@ -1,20 +1,22 @@
 import React from 'react'
 import T from 'prop-types'
 import { List, ListItem, ListItemText } from '@material-ui/core'
+import { useTheme } from '@material-ui/core/styles'
 import startCase from 'lodash.startcase'
 
 import useFields from '../hooks/useFields'
 
 export default function EntityFields({ typeName, data }) {
   const fields = useFields(typeName)
+  const theme = useTheme()
 
   return (
     <List>
       {fields.all.map(field => (
         <ListItem key={field}>
           <ListItemText
             primary={startCase(field)}
-            secondary={fields.format(field, data[field])}
+            secondary={fields.format(field, data[field], theme)}
```

Comment:
.format is a method of fields -> fields comes from useFields -> useFields is a hook. why don't we useTheme within useFields instead of providing the theme to .format?

Reply:
Yep! I tried something similar, but it was late and head was foggy 😂
This was easy enough to change.